tensorflow/contrib/framework/python/ops/variables.py

```diff
@@ -50,9 +51,29 @@
            'local_variable',
            'model_variable',
            'variable',
-           'VariableDeviceChooser']
+           'VariableDeviceChooser',
+           'zero_initializer']
 
 
+def zero_initializer(ref, use_locking=True, name="zero_initializer"):
```

Comment:
Is this wrapper necessary? I think we should be able to just forward the generated op.